tools/trace-plotter/loader/zipkinApiLoader.go

```diff
@@ -0,0 +1,57 @@
+package loader
```

Comment:
does it make sense to have 2 ways? I'd go with the simpler or more practical way and remove the alternative